src/brsTypes/components/RoDeviceInfo.ts

```diff
@@ -0,0 +1,687 @@
+        signature: {
+            args: [],
+            returns: ValueKind.Boolean,
+        },
+        impl: (_interpreter) => {
+            return BrsBoolean.True;
+        },
+    });
+
+    private getChannelClientId = new Callable("getChannelClientId", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            return new BrsString("123e4567-e89b-12d3-a456-426655440000");
+        },
+    });
+
+    private getUserCountryCode = new Callable("getUserCountryCode", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            let countryCode = process.env.LOCALE;
+            return countryCode ? new BrsString(countryCode) : BrsInvalid.Instance;
+        },
+    });
+
+    private getRandomUUID = new Callable("getRandomUUID", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            return new BrsString("123e4567-e89b-12d3-a456-426655440000");
+        },
+    });
+
+    private getTimeZone = new Callable("getTimeZone", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            let timeZone = process.env.TZ;
+            return timeZone ? new BrsString(timeZone) : BrsInvalid.Instance;
+        },
+    });
+
+    private hasFeature = new Callable("hasFeature", {
+        signature: {
+            args: [new StdlibArgument("feature", ValueKind.String)],
+            returns: ValueKind.Boolean,
+        },
+        impl: (_interpreter) => {
+            return BrsBoolean.True;
+        },
+    });
+
+    private getCurrentLocale = new Callable("getCurrentLocale", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            let locale = process.env.LOCALE;
+            return locale ? new BrsString(locale) : BrsInvalid.Instance;
+        },
+    });
+
+    private getCountryCode = new Callable("getCountryCode", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            let countryCode = process.env.LOCALE;
+            return countryCode ? new BrsString(countryCode) : BrsInvalid.Instance;
+        },
+    });
+
+    private getPreferredCaptionLanguage = new Callable("getPreferredCaptionLanguage", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            return new BrsString("eng");
+        },
+    });
+
+    private timeSinceLastKeyPress = new Callable("timeSinceLastKeyPress", {
+        signature: {
+            args: [],
+            returns: ValueKind.Int32,
+        },
+        impl: (_interpreter) => {
+            return new Int32(360);
```

Comment:
Maybe let's return 0 here? 0 is a little more predictable/"default" than anything else.